commander/src/base_bootstrap_command.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { Command, flags as flagParser } from '@oclif/command';
+
+interface BootstrapFlags {
+	readonly template: string;
+}
+
+export default abstract class BaseBootstrapCommand extends Command {
+	static flags = {
```

Comment:
if it's just a flag, does it needs to be base command? I think it's ok to just use the common flag variable

Reply:
Along with flag there is parsing logic and some other code will come which is useful for all bootstrpping commands so I think its good to keep it this way. We also used similar approach on other places.